assets/src/edit-story/components/panels/font.js

```diff
@@ -106,39 +129,79 @@ function FontPanel({ selectedElements, onSetProperties }) {
+                  ? state.fontWeight
+                  : 400;
+              setState({
+                ...state,
+                fontFamily: value,
+                fontWeight: parseInt(newFontWeight),
+                fontWeights: currentFontWeights,
+                fontFallback: currentFontFallback,
+              });
+            }}
+          />
+        )}
+      </Row>
+      <Row>
         <SelectMenu
-          label={__('Font family', 'web-stories')}
-          options={fonts}
-          value={state.fontFamily}
-          isMultiple={fontFamily === ''}
-          onChange={(value) => {
-            const currentFontWeights = getFontWeight(value);
-            const currentFontFallback = getFontFallback(value);
-            const fontWeightsArr = currentFontWeights.map(
-              ({ thisValue }) => thisValue
-            );
-            const newFontWeight =
-              fontWeightsArr && fontWeightsArr.includes(state.fontWeight)
-                ? state.fontWeight
-                : 400;
-            setState({
-              ...state,
-              fontFamily: value,
-              fontWeight: parseInt(newFontWeight),
-              fontWeights: currentFontWeights,
-              fontFallback: currentFontFallback,
-            });
-          }}
+          ariaLabel={__('Font style', 'web-stories')}
+          options={fontStyles}
+          isMultiple={fontStyle === ''}
+          value={state.fontStyle}
+          onChange={(value) => setState({ ...state, fontStyle: value })}
         />
-      )}
-      <SelectMenu
-        label={__('Font style', 'web-stories')}
-        options={fontStyles}
-        isMultiple={fontStyle === ''}
-        value={state.fontStyle}
-        onChange={(value) => setState({ ...state, fontStyle: value })}
-      />
+        <Space />
+        <BoxedNumeric
+          ariaLabel={__('Font size', 'web-stories')}
+          value={state.fontSize}
+          isMultiple={fontSize === ''}
+          min={MIN_FONT_SIZE}
+          max={MAX_FONT_SIZE}
+          flexBasis={58}
+          textCenter
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
+          }
+        />
+      </Row>
+      <Row>
+        {/* TODO: Add vertical offset logic */}
+        <ExpandedNumeric
+          ariaLabel={__('Vertical offset', 'web-stories')}
+          value={state.fontSize}
+          suffix={<VerticalOffset />}
+          isMultiple={fontSize === ''}
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
+          }
+        />
+        <Space />
+        {/* TODO: Add horizontal offset logic */}
+        <ExpandedNumeric
+          ariaLabel={__('Horizontal offset', 'web-stories')}
+          value={state.fontSize}
+          suffix={<HorizontalOffset />}
+          isMultiple={fontSize === ''}
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
```

Comment:
This is probably for `letterSpacing` (which should already be functional, too), in another panel though (the `textStyle` panel).

I'm happy to address the functionality of this in the Design Panel functionality PRs instead though.